apps/src/sites/studio/pages/teacher_dashboard/index.js

```diff
@@ -2,22 +2,63 @@ import $ from 'jquery';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import {BrowserRouter as Router, Route} from 'react-router-dom';
+import {Provider} from 'react-redux';
+import {getStore, registerReducers} from '@cdo/apps/redux';
+import manageStudents, {
+  setLoginType,
+  setStudents,
+  convertStudentServerData,
+  toggleSharingColumn,
+} from '@cdo/apps/templates/manageStudents/manageStudentsRedux';
+import teacherSections, {
+  asyncLoadSectionData
+} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import sectionData, {setSection} from '@cdo/apps/redux/sectionDataRedux';
 import TeacherDashboard from '@cdo/apps/templates/teacherDashboard/TeacherDashboard';
 
 const script = document.querySelector('script[data-dashboard]');
 const scriptData = JSON.parse(script.dataset.dashboard);
-const sectionId = scriptData.section_id;
-const baseUrl = `/teacher_dashboard/sections/${sectionId}`;
+const section = scriptData.section;
+const baseUrl = `/teacher_dashboard/sections/${section.id}`;
 
 $(document).ready(function () {
+  registerReducers({teacherSections, manageStudents, sectionData});
+  const store = getStore();
+  store.dispatch(setLoginType(section.login_type));
+  store.dispatch(asyncLoadSectionData(section.id));
```

Comment:
i am actually refactoring this method (`asyncLoadSectionData`) to live in sectionDataRedux as part of the stats table work i'm doing -- it feels like it makes more sense there, since that's what sectionDataRedux was built to do... what do you think?

Reply:
I think that makes sense! Here I mostly lifted wholesale from [section.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/teacherDashboard/sections.js); anything we can do to clean up as we convert seems like a great choice. Do you want me to leave this here for now or change to match your re-factor?

Reply:
you can leave it as-is and i'll make sure to wait until this is merged and will pull it in to my refactor